### PR TITLE
Extend createTransactionAsset with uploadPlatform

### PR DESCRIPTION
**Pull request by amodes**
Addresses
https://kontist.atlassian.net/jira/software/c/projects/KRP/boards/20?modal=detail&selectedIssue=KRP-814&assignee=70121%3Ab6b8c151-f2e4-49ef-acbc-b40b08a85f0e

Related BE PR:
https://github.com/kontist/backend/pull/12041

### CODE REVIEW

File: lib/graphql/schema.ts [hunk 1]
```diff
@@ -1171,6 +1171,7 @@ export type MutationCreateTransactionAssetArgs = {
   filetype: Scalars['String'];
   name: Scalars['String'];
   transactionId: Scalars['ID'];
+  uploadPlatform?: InputMaybe<Scalars['String']>;
```
Comment (p-janik) [inline lib/graphql/schema.ts:1174]: don't we have an enum for that?

Comment (p-janik) [reply lib/graphql/schema.ts:1174]: I have proper type when i do:
```
@Arg("uploadPlatform", (type) => RequestPlatform, { nullable: true })
    uploadPlatform?: RequestPlatform
```

Comment (amodes) [reply lib/graphql/schema.ts:1174]: Adjusted BE here: https://github.com/kontist/backend/pull/12046

**Review by p-janik — APPROVED** [review]
LGTM, let's just update type after backend is ready

**Comment by UgurGumushan** [comment]
Hello;
I've made [a release](https://github.com/kontist/js-sdk/releases/tag/v0.47.2) which is going to work with the backend after the release of TSB-11 (multiple projects are involved).
If we merge this PR to master and set the backend to use that new version, the tests on the backend would fail. 
I need some time to complete TSB-11 as well, so if you need this feature early, we can release it without merging it to master, it could be 0.46.3, released from this branch
Any ideas?

**Comment by p-janik** [comment]
I think there is no rush with releasing this one

**Pull request merged**